test/api-example-generator.test.js

```diff
@@ -328,10 +328,10 @@ describe('<api-example-generator>', () => {
           const result = element.computeExamples(shape, 'application/json');
           assert.typeOf(result, 'array');
           assert.lengthOf(result, 1);
-          assert.isFalse(result[0].hasRaw);
```

Comment:
This is obviously major change in the component's logic. Meaning there's something wrong. This should be `false` and it isn't. The change happened after changing AMF's version and not component's logic.

Reply:
To be changed to consider empty arrays as not raw